apps/server/src/modules/tldraw/controller/tldraw.controller.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('TldrawController', () => {
 					docName: 'test-name',
 				};
 
-				const ucSpy = jest.spyOn(service, 'deleteByDocName').mockImplementation(() => Promise.resolve());
+				const ucSpy = jest.spyOn(service, 'deleteByDocName').mockResolvedValueOnce();
```

Comment:
the spy is not needed. All functions of mockobjects are spies already. you could just do 
```
const setup = () => {
	const params: TldrawDeleteParams = {
		docName: 'test-name',
	};

	return { params };
};

it('should call service with parentIds', async () => {
	const { params } = setup();
	await controller.deleteByDocName(params);
	expect(service.deleteByDocName).toHaveBeenCalledWith('test-name');
});
```

On the other hand, this test is probably not really needed anyway, as there should be an api test for this endpoint instead (which appears to be missing)

Reply:
as discussed, not in the scope of this PR so will be done in a separate one